libvast/CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ set(libvast_sources
     src/ewah_bitmap.cpp
     src/expression.cpp
     src/expression_visitors.cpp
+    src/fbs/utils.cpp
```

Comment:
This file is not added in the same commit. Maybe I'll discover it later?

Reply:
Yeah, that happened when I factored my longer commit into multiple smaller ones.